ASM/c/main.c

```diff
@@ -55,6 +55,7 @@ void after_game_state_update() {
         draw_dungeon_info(&(z64_ctxt.gfx->overlay));
         draw_triforce_count(&(z64_ctxt.gfx->overlay));
         draw_silver_rupee_count(&z64_game, &(z64_ctxt.gfx->overlay));
+        draw_boss_key(&z64_game, &(z64_ctxt.gfx->overlay));
```

Comment:
Shouldn't this be called before `draw_silver_rupee_count` so the position of the silver rupee count doesn't lag behind by a frame?

Reply:
In fact, you can see a similar issue with the small key count:

![image](https://github.com/OoTRandomizer/OoT-Randomizer/assets/641386/c249ddb4-036a-4552-aa15-0eea900c4d3a)

Would it be possible to move the `is_bk_displayed` call to somewhere earlier in the frame to prevent this?